rtengine/camconst.json

```diff
@@ -2263,6 +2263,14 @@ Camera constants:
         }
     },
 
+    { // Quality C
+        "make_model": "Panasonic DC-G9M2",
+        "dcraw_matrix": [ 8325, -3456, -623, -4330, 12089, 2528, -860, 2646, 5984 ], // Adobe DNG Converter
+        "ranges": {
+            "white": 65535
+        }
```

Comment:
We should see why the decoder does not have the white level for this camera.

Reply:
The issue is in `dcraw.cc` `DCraw::adobe_coeff`. There's an entry for a `Panasonic DC-G9` camera and this function uses a prefix so it matches also `Panasonic DC-G9M2`.
https://github.com/Beep6581/RawTherapee/blob/5a25619bd50d42059abe9818420f639b6d10e3bf/rtengine/dcraw.cc#L8892-L8893

But this camera has more detailed information provided in camconst so this entry could just be removed from dcraw:adobe_coeffs since it's overridden by camconst entries. This will also fix this issue.

@Lawrence37 Is it ok for you? Should I create a new PR or a new commit in this PR to remove this entry from `DCraw::adobe_coeff`?

Reply:
I prefer to keep the dcraw value there because the decoder should provide a reasonable white level in the absence of a camconst entry. We should add the DC-G9M2 values from Libraw:
https://github.com/LibRaw/LibRaw/blob/158e635e5e80a95dfa402b0a79a7167922c1d4ec/src/tables/colordata.cpp#L1367-L1371

Reply:
@Lawrence37 I you think it'll be better I'll do this but I'm quite confused why `adobe_coeffs` is better than camconst in this scenario. I thought camconst was the new rawtherapee way to handle camera properties and adobe_coeffs the old dcraw way before camconst were introduced (I also thought that `adobe_coeffs` table could be removed and migrated to camconst but I may have lost something).

Reply:
One is not necessarily better than the other. They serve different purposes. `adobe_coeff` provides constants straight out of the decoder. `camconst.json` is for overriding, fine-tuning, or supplementing the decoder's values. It's like coding style. There's no difference for the user, but it improves maintainability for developers.